### ISSUE TITLE
Length of group in Range(Fibonacci)

### ISSUE THREAD

**Issue by kgs** [InteractiveAdvertisingBureau/Global-Privacy-Platform#68]
I want to understand what is the meaning of "length" field. In example from specification, start of the group is 5 with length equal to 3. I was expecting range of 3 values (5, 6, 7) but there are 4 in this example (5, 6, 7, 8). Which case is correct?
![image](https://user-images.githubusercontent.com/41653/229122977-0be9d4eb-7813-4e63-b4bc-e327eee01b3d.png)

**Comment by janwinkler** [comment]
yes, "length" is not the best word here since it does not give the length of the group but the "distance" between first and last ID

first item:
- single item of 3

second item:
- group starting at 2 IDs away from last item's end (3), hence start = 5
- group end has a distance of 3, therefore the last ID is 8
- group contains the IDs 5 to 8 (= 5,6,7,8)
